### PR TITLE
Added "Monthly App Store Review Process Page" to Doc Site

### PR DESCRIPTION
**Pull request by brea11y**
## Description of Change
I added a new page "Monthly App Store Review Process" page to the doc site. This page will be nested under the research tab, which required some restructuring of that area to accommodate the change.

## Testing

- [ ] Tested on iOS 
- [ ] Tested on Android 

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [ ] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

**Review by theodur — DISMISSED** [review]

File: VAMobile/documentation/docs/UX/Foundations/Research/montly-reviews.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: Monthly App Store Reviews
+sidebar_position: 2
+---
+
+# Monthly App Store Review Process
+
+## Step 1: Download the reviews from the [feedback hub](https://feedback-hub-e659c24714b9.herokuapp.com/)
+1. Select App Store or Play Store at the top.
```
Comment (timwright12) [inline VAMobile/documentation/docs/UX/Foundations/Research/montly-reviews.md:9]:
```suggestion

1. Select App Store or Play Store at the top.
```

Comment (timwright12) [reply VAMobile/documentation/docs/UX/Foundations/Research/montly-reviews.md:9]: Lists should have a space above and below for markdown rendering

File: VAMobile/documentation/docs/UX/Foundations/Research/montly-reviews.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+3. Input the date range for the month you are working on.
+4. Tap “search”.
+5. Click the “Download as .csv” button to export.
+
+## Step 2: Import the data to Google Sheets 
+[In the spreadsheet](https://docs.google.com/spreadsheets/d/1Lj65hvWUkdTpmpvsObOOD2hjG8CHAiQ7z0mCeowabh0/edit?usp=sharing), create two new sheets for the month (one for overview and one for reviews) and import the reviews for the corresponding date into that month’s "reviews" sheet.
+- _Note: You can duplicate a previous month and remove the data._
+
+## Step 3: Clean up the import
+1. Verify that data pulled in correctly
```
Comment (timwright12) [inline VAMobile/documentation/docs/UX/Foundations/Research/montly-reviews.md:20]:
```suggestion

1. Verify that data pulled in correctly
```

File: VAMobile/documentation/docs/UX/Foundations/Research/montly-reviews.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+2. Add wrapping to the grids
+3. Verify that the conditional formatting for the ratings pulled in (you should see red, green, etc. in the column).
+    - If not, apply it to the column.
+4. Define the range for the tags column (i.e. TagMarch24). 
+    - This allows the spreadsheet to do the math and pull the totals into the overview sheet.
+
+## Step 4: Analyze the data
+1. Update the overview with the correct dates and review totals for the month.
```
Comment (timwright12) [inline VAMobile/documentation/docs/UX/Foundations/Research/montly-reviews.md:28]:
```suggestion

1. Update the overview with the correct dates and review totals for the month.
```

File: VAMobile/documentation/docs/UX/Foundations/Research/montly-reviews.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+5. Pull out a few of the reviews that you think are important and add them to the review sample column (try to limit the samples to 2-4).
+    - If the trending sentiment is “mixed”, try to pull both positive and negative samples into the overview.
+
+## Step 5: Find or create a new folder on Github
+In the [App Store Reviews folder on Github](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/va-mobile-app/reporting/App-Stores-Reviews), find or create the folder for the corresponding quarter.
+
+If you are adding app store reviews for a month in an existing quarter’s folder, select the month that you are updating and edit the file.
+
+### How to create a folder for a new quarter
+1. Create the new folder (add a file in Github and `/` at the end to create a new folder).
```
Comment (timwright12) [inline VAMobile/documentation/docs/UX/Foundations/Research/montly-reviews.md:45]:
```suggestion

1. Create the new folder (add a file in Github and `/` at the end to create a new folder).
```

File: VAMobile/documentation/docs/UX/Foundations/Research/montly-reviews.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    - Follow the naming convention of the previous quarters (four digit year followed by a hyphen and then the quarter).
+2. Create placeholder files for all three months in the quarter. 
+    - Follow the naming convention of the other files (i.e. 2024-04-Summary-VA-Mobile-App-Store-Reviews.md)
+3. Add a readme.md file to the folder (content can be copied from the readme.md file in another folder, but be sure to update the dates in the file to the new quarter).
+        
+## Step 6: Add the overview to Github
+1. Back in your app store review sheet, select only your table in the overview and then [convert the overview table to markdown](https://tabletomarkdown.com/convert-spreadsheet-to-markdown/).
```
Comment (timwright12) [inline VAMobile/documentation/docs/UX/Foundations/Research/montly-reviews.md:52]:
```suggestion

1. Back in your app store review sheet, select only your table in the overview and then [convert the overview table to markdown](https://tabletomarkdown.com/convert-spreadsheet-to-markdown/).
```

**Comment by brea11y** [comment]
@timwright12 – I approved all of your suggested updates last week. Is it possible to get this PR approved and pushed through? Thank you!

**Review by timwright12 — APPROVED** [review]

**Pull request merged**